node-runner-cli/templates/radix-fullnode-compose.yml.j2

```diff
@@ -51,7 +52,7 @@ services:
     {% if core_node.data_directory is defined %}
     - babylon_ledger:/home/radixdlt/RADIXDB
     {% endif %}
-    - {{core_node.keydetails.keyfile_path ~ '/' ~ core_node.keydetails.keyfile_name }}:/home/radixdlt/node-keystore.ks
+    - {{core_node.keydetails.keyfile_path ~ '/' }}:/home/radixdlt/babylon-node-config/
```

Comment:
Just thinking again, this take away the custom name of keystore file. Why not map the exact file instead of directory? that way keystore if named differently will still get mounted

Reply:
yeah I think that was precisely my thinking, so if we mounted the directory, the ks will always be mounted, independently of it's name no?